diff --git a/configure b/configure
index 5eba632..75a8676 100755
--- a/configure
+++ b/configure
@@ -1857,6 +1857,7 @@ if test -z "$_target" ; then
       nios2) host_arch=nios2 ;;
       vax) host_arch=vax ;;
       xtensa*) host_arch=xtensa ;;
+      loongarch64) host_arch=loongarch64 ;;
       *) host_arch=UNKNOWN ;;
   esac
   }
@@ -2850,6 +2851,18 @@ EOF
     arch='arc'
     iproc='arc'
     ;;
+  loongarch64)
+    arch='loongarch'
+    subarch='loongarch64'
+    def_fast_unaligned='#define HAVE_FAST_UNALIGNED 1'
+    def_av_fast_unaligned='#define AV_HAVE_FAST_UNALIGNED 1'
+    def_local_aligned_8='#define HAVE_LOCAL_ALIGNED_8 1'
+    def_local_aligned_16='#define HAVE_LOCAL_ALIGNED_16 1'
+    def_local_aligned_32='#define HAVE_LOCAL_ALIGNED_32 1'
+    def_fast_64bit='#define HAVE_FAST_64BIT 1'
+    _march='-march=loongarch'
+    iproc='loongarch'
+    ;;
 
   *)
     echo "The architecture of your CPU ($host_arch) is not supported by this configure script"
diff --git a/stream/stream_smb.c b/stream/stream_smb.c
index 53fd821..824848e 100644
--- a/stream/stream_smb.c
+++ b/stream/stream_smb.c
@@ -18,7 +18,7 @@
 
 #include "config.h"
 
-#include <libsmbclient.h>
+#include <samba-4.0/libsmbclient.h>
 #include <unistd.h>
 
 #include "mp_msg.h"
