--- a/Makefile	2023-12-25 22:47:19.080000000 +0800
+++ b/Makefile	2023-12-25 22:50:35.160000000 +0800
@@ -83,7 +83,7 @@ PKG=github.com/containerd/containerd

 # Project binaries.
 COMMANDS=ctr containerd containerd-stress
-MANPAGES=ctr.8 containerd.8 containerd-config.8 containerd-config.toml.5
+MANPAGES=

 ifdef BUILDTAGS
     GO_BUILDTAGS = ${BUILDTAGS}
@@ -149,7 +149,7 @@ GOTEST ?= $(GO) test
 OUTPUTDIR = $(join $(ROOTDIR), _output)
 CRIDIR=$(OUTPUTDIR)/cri

-.PHONY: clean all AUTHORS build binaries test integration generate protos check-protos coverage ci check help install uninstall vendor release static-release mandir install-man genman install-cri-deps cri-release cri-cni-release cri-integration install-deps bin/cri-integration.test
+.PHONY: clean all AUTHORS build binaries test integration generate protos check-protos coverage ci check help install uninstall vendor release static-release mandir install-cri-deps cri-release cri-cni-release cri-integration install-deps bin/cri-integration.test
 .DEFAULT: default

 # Forcibly set the default goal to all, in case an include above brought in a rule definition.
@@ -254,11 +254,6 @@ endef
 bin/%: cmd/% FORCE
 	$(call BUILD_BINARY)

-# gen-manpages must not have the urfave_cli_no_docs build-tag set
-bin/gen-manpages: cmd/gen-manpages FORCE
-	@echo "$(WHALE) $@"
-	$(GO) build ${DEBUG_GO_GCFLAGS} ${GO_GCFLAGS} ${GO_BUILD_FLAGS} -o $@ ${GO_LDFLAGS} $(subst urfave_cli_no_docs,,${GO_TAGS})  ./cmd/gen-manpages
-
 bin/containerd-shim: cmd/containerd-shim FORCE # set !cgo and omit pie for a static shim build: https://github.com/golang/go/issues/17789#issuecomment-258542220
 	@echo "$(WHALE) $@"
 	@CGO_ENABLED=${SHIM_CGO_ENABLED} $(GO) build ${GO_BUILD_FLAGS} -o $@ ${SHIM_GO_LDFLAGS} ${GO_TAGS} ./cmd/containerd-shim
@@ -280,17 +275,6 @@ man: mandir $(addprefix man/,$(MANPAGES)
 mandir:
 	@mkdir -p man

-# Kept for backwards compatibility
-genman: man/containerd.8 man/ctr.8
-
-man/containerd.8: bin/gen-manpages FORCE
-	@echo "$(WHALE) $@"
-	$< $(@F) $(@D)
-
-man/ctr.8: bin/gen-manpages FORCE
-	@echo "$(WHALE) $@"
-	$< $(@F) $(@D)
-
 man/%: docs/man/%.md FORCE
 	@echo "$(WHALE) $@"
 	go-md2man -in "$<" -out "$@"
