--- a/libcontainer/init_linux.go	2023-12-23 13:45:16.976000000 +0800
+++ b/libcontainer/init_linux.go	2023-12-23 13:45:37.148000000 +0800
@@ -381,10 +381,10 @@ func setupUser(config *initConfig) error
 		}
 	}
 
-	if err := system.Setgid(execUser.Gid); err != nil {
+	if err := unix.Setgid(execUser.Gid); err != nil {
 		return err
 	}
-	if err := system.Setuid(execUser.Uid); err != nil {
+	if err := unix.Setuid(execUser.Uid); err != nil {
 		return err
 	}
 
